src/module/actor/party/document.ts

```diff
@@ -255,6 +255,11 @@ class PartyPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e | n
         options: PartyUpdateOperation<TParent>,
         user: UserPF2e,
     ): Promise<boolean | void> {
+        // Prevent party actors from being dragged to folders
+        if (changed.folder) {
+            changed.folder = null;
+        }
```

Comment:
```suggestion
        changed.folder &&= null;
```

Reply:
Not quite the same thing, but it also works and is one line.